ui/widgets/datepicker.js

```diff
@@ -79,30 +81,33 @@ var widget = $.widget( "ui.datepicker", {
 	_getCreateOptions: function() {
 		var max = this.element.attr( "max" ),
 			min = this.element.attr( "min" ),
+			parser = new Globalize( "en" ).dateParser( { raw: "yyyy-MM-dd" } ),
```

Comment:
For this raw pattern, the locale might not be required, since there's nothing locale-specific, I think. In general, Globalize doesn't bundle any locales.